extensions/mesa-vpu.sh

```diff
@@ -72,6 +78,36 @@ function post_install_kernel_debs__3d() {
 		Pin-Priority: 1001
 		EOF
 
+		if [[ "${ARCH}" == "arm64" ]]; then
+
+			display_alert "Adding Amazingfate Chromium PPAs" "${EXTENSION}" "info"
+			do_with_retries 3 chroot_sdcard add-apt-repository ppa:liujianfeng1994/chromium --yes --no-update
+			sed -i "s/oracular/noble/g" "${SDCARD}"/etc/apt/sources.list.d/liujianfeng1994-ubuntu-chromium-"${RELEASE}".*
+
+			display_alert "Pinning amazingfated's Chromium PPAs" "${EXTENSION}" "info"
+			cat <<- EOF > "${SDCARD}"/etc/apt/preferences.d/liujianfeng1994-chromium-pin
+			Package: *
```

Comment:
I think we should just pin chromium. The PPA also has some extra packes like linux, llvm toolchain etc.

Reply:
I'm planning to remove build only deps like llvm and use a separate dep ppa to manage these build deps.